.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistency
+# coding styles between different editors and IDEs
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{md,mdx}]
+indent_size = 4
+trim_trailing_whitespace = false
+max_line_length = 80
+
+[*.{html,vue,css,less,sass,scss,js,ts,jsx,tsx,mjs}]
+indent_size = 4
+max_line_length = 80
```

Comment:
no max line length

Reply:
use web standard line length

Reply:
This is web standard line height for html/css